### DISCUSSION TITLE
Avalonia.Win32 0.10.5 is not compatible with net5

### DISCUSSION THREAD

**Discussion by szekelymatyas** [AvaloniaUI/Avalonia#6005] [Q&A]
Today i made a empty avalonia project with VS2019 16.10 and the lates extension. The avalonia version is 0.10.3 when i try to update the nugets, i got this message:

Package Avalonia.Win32 0.10.5 is not compatible with net5.0 (.NETCoreApp,Version=v5.0). Package Avalonia.Win32 0.10.5 does not support any target frameworks.

Any idea how to solve this?

**Comment by RomanSoloweow** [comment]
the same problem

**Comment by aetaylor** [comment]
Same here.

**Comment by kekekeks** [comment]
Windows defender has probably deleted it.
https://github.com/AvaloniaUI/Avalonia/issues/6009

Remove Avalonia.Win32 from your nuget cache and reinstall
